pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -48,6 +48,8 @@ const (
 	udevRulesDir       = "/etc/udev/rules.d"
 	udevRpsRule        = "99-netdev-rps"
 	setRPSMask         = "set-rps-mask"
+	RunCWrapper        = "runc-wrapper"
+	runCWrapperCPUMask = "CPUMask"
```

Comment:
Maybe we want to change this to CPUList (optional)

Reply:
ok